From a0e761ccaac0f623732a5f493a3d1979b0807094 Mon Sep 17 00:00:00 2001
From: Carsten Haitzler (Raster <raster@rasterman.com>
Date: Wed, 29 Oct 2008 21:03:52 +1100
Subject: [PATCH] Linux (Palm): Fix Linux kernel build so it actually works for palmt650

Linux hackndev changed their git so the OE packages for palm ceased to work.
This updates so it actually builds a kernel corectly.
---
 packages/linux/linux-hackndev-2.6_git.bb |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/packages/linux/linux-hackndev-2.6_git.bb b/packages/linux/linux-hackndev-2.6_git.bb
index 71e1235..015a6fe 100644
--- a/packages/linux/linux-hackndev-2.6_git.bb
+++ b/packages/linux/linux-hackndev-2.6_git.bb
@@ -20,6 +20,7 @@ PV = "${K_MAJOR}.${K_MINOR}.${K_MICRO}-${HHV}"
 inherit kernel
 
 do_configure() {
+	echo "working in $PWD"
 
 	if [ -f ${WORKDIR}/defconfig ]; then
 		install -m 0644 ${WORKDIR}/defconfig ${S}/.config || die "No configuration for ${MACHINE} available."
-- 
1.5.4.3

